cluster/mesos/cluster.go

```diff
@@ -169,31 +167,31 @@ func NewCluster(scheduler *scheduler.Scheduler, TLSConfig *tls.Config, master st
 
 // Handle callbacks for the events
 func (c *Cluster) Handle(e *cluster.Event) error {
```

Comment:
This sounds like a reasonable thing to do. I believe in addition to renaming `HandleAll`, we'll also need to change the interface to not have `Handle` return the error, as you said above.